hosting/nginx.prod.conf.hbs

```diff
@@ -117,12 +117,21 @@ http {
     }
 
     location /api/backups/ {
+      # calls to export apps are limited
```

Comment:
Could just update this to be more general, as it affects all backup endpoints and not just the export